example/index.html

```diff
@@ -189,6 +189,11 @@
         accessors: {
           icon: 'nodeType'
         },
+        label: function (selection) {
```

Comment:
I'm trying to think through this.  So what else would make sense to change on the selection aside from the text/html?  I guess you could add additional CSS classes.

If the only thing that would make sense to customize is whether or not to encode the output, it would be simpler to just pass in a boolean flag to the constructor.  If there are other interesting things we might want to change here, then maybe a boolean would be too simplistic.